docs/semgrep-app/integrations.md

```diff
@@ -1,19 +1,163 @@
+
+![Screenshot of the Semgrep rule board with integrations](../img/integration-ruleboard.png)<br />
+
+## Integrating various third-party tools
+
+This section describes how to integrate Semgrep App into particular third-party tools.
+
+### Slack
+
+Integrating with Slack allows Semgrep to send findings to a channel on your Slack workspace.
+
+To set it up, create an app within Slack and enable that app's **Incoming Webhooks** feature. By creating a webhook, you'll have a URL that serves as Semgrep's endpoint to your Slack workplace. Follow the guide below to create the app and webook, and integrate with Semgrep:
+
+1. Log in to your Slack account. Enter your login details in [Slack workspace](https://slack.com/workspace-signin).
+2. Create an app in your Slack workplace by following this link: [Slack apps](https://api.slack.com/apps?new_app=1).
+3. Select **From scratch** and fill in the fields for your app's name and workspace.
+![Screenshot of a Slack form to fill out new app details](../img/slack-create-app-name.png)
+4. You will be redirected to your new app's settings page. Alternatively, you can find this page by viewing your apps [here](https://api.slack.com/apps) and selecting your integration app from the menu.
+5. Click on **Incoming Webhooks** on the left menu, and then enable the toggle.
+6. After you have enabled **Incoming Webhooks**, follow one of these steps:
+    - Generate a **Webhook URL**.
+    - Request to add a new webhook from a workspace owner.
+7. After you've obtained the Webhook URL, return to Semgrep and create a new Slack integration. Name the integration and copy-paste the URL into the **Slack URL** field. You may opt to select the **Inventory** flag to include notifications about Code Asset Inventory findings.
+8. You can **Test** then select which channel you'd like Semgrep to start posting to.
+![Screenshot of the Slack Request Webhook  ](../img/slack-integration-hooktest.png)<br />
+9. After you've verified that everything is working, remember to **Save** the final configuration.
+10. Remember to turn notifications on by going to the **Rule board**, clicking on the **gear icon,** then clicking on the toggle.
+
+Here is a sample of a Slack message sent from Semgrep with findings:
+
+![Screenshot of Slack message with findings ](../img/slack-semgrep-findings.png)<br />
+
+#### Additional resources
+
+* https://api.slack.com/apps
+* https://api.slack.com/messaging/webhooks#enable_webhooks
+
+#### See also
+
+[Notifications -> Slack](notifications.md/#slack)
+
+### Email
+
+Email integration allows Semgrep to send new findings to an email address of your choice.
+
+To set up email integration:
+
+1. In **Integrations,** click **Add Integration.**
+2. Click on **Email**.
+3. Enter a **Name** for the integration.
+4. Enter the **Email address** that will receive Semgrep findings.
+5. Select the **Inventory** check box if you would like to receive notifications about Code Asset Inventory findings.
```

Comment:
```suggestion
5. Optional: Select the **Inventory** check box to receive notifications about Code Asset Inventory findings.
```

Avoid modal verbs. To make this step optional, mark it as optional. See: https://developers.google.com/style/procedures?hl=en#optional-steps